charts/catalog/README.md

```diff
@@ -54,11 +57,23 @@ chart and their default values.
 | `apiserver.storage.etcd.persistence.size` | PVC Storage Request | `4Gi` |
 | `apiserver.verbosity` | Log level; valid values are in the range 0 - 10 | `10` |
 | `apiserver.auth.enabled` | Enable authentication and authorization | `true` |
+| `apiserver.audit.activated` | If true, enables the use of audit features via this chart. | `false` |
+| `apiserver.audit.logPath` | If specified, audit log goes to specified path. | `"/tmp/service-catalog-apiserver-audit.log"` |
+| `apiserver.serviceAccount` | Service account. | `service-catalog-apiserver` |
+| `apiserver.serveOpenAPISpec` | If specified, audit log goes to specified path. | `"/tmp/service-catalog-apiserver-audit.log"` |
```

Comment:
This should be "if true, makes the API server serve the OpenAPI schema"

Reply:
@carolynvs Done, sorry about that! Please have another look, thanks!